From 227d01e985356ada6097de1077309054ca3a8c1a Mon Sep 17 00:00:00 2001
From: Dirk Hohndel <dirk@hohndel.org>
Date: Thu, 3 Dec 2020 15:01:18 -0800
Subject: [PATCH 1/2] include the shared / not shared result in log

We shouldn't be building Kirigami as shared library, yet that's
what makes our GitHub builds fail.

Signed-off-by: Dirk Hohndel <dirk@hohndel.org>
---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 65349a48..d75f17a4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,9 +52,12 @@ set(AUTOMOC_MOC_OPTIONS -Muri=org.kde.kirigami)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 if(NOT BUILD_SHARED_LIBS)
+    message(STATUS "Static build of libkirigami")
     add_definitions(-DKIRIGAMI_BUILD_TYPE_STATIC)
     add_definitions(-DQT_PLUGIN)
     add_definitions(-DQT_STATICPLUGIN=1)
+else()
+    message(STATUS "Shared build of libkirigami -- BUILD_SHARED_LIBS is \"${BUILD_SHARED_LIBS}\"")
 endif()
 
 ################# set KDE specific information #################
-- 
2.51.1

